rancher/distros-test-framework
# Update install scripts and modules for dual stack support

#### Proposed Changes ####


- Update install scripts and terraform files in modules to support dual stack
- Replace local vars with global vars in install scripts
- Organize and Add consistency across install scripts for the products 



#### Types of Changes ####


- Refactor for dual stack tests

#### Testing ####

- Run through existing jobs on the branch to ensure no regression is observed

Checklist:
1. If your PR changes anything on or related to Jenkins, run it pointing to your branch to make sure it's okay.
- Ok

2. Verify code lint; we should not have errors.
- Ok

3. Update the documentation if needed.
- N/A

4. Update makefile and docker run if adding new tests.
- N/A

5. Run your tests at least 4 times with all configurations needed and possible.
- Ok

6. If needed test with different os types.
- Ok

#### Linked Issues ####


https://github.com/rancher/qa-tasks/issues/882

#### Further Comments ####



GH Actions for linting: https://github.com/mdrahman-suse/distros-test-framework/actions
